lib/takeaway_interface.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "menu"
+require_relative "summary"
+require_relative "text"
+require_relative "order"
+
+module TakewayInterface
+  class Takeway
+
+    def initialize(menu = Menu.new)
+      @menu = menu
+      @order = nil
+    end
+
+    def run
```

Comment:
correct me if i'm wrong, this module, provides a schedule of sorts for the menu? 
Weather i'm wrong about the functionality or not (i probably am) could this have been worked into the class itself, or vice versa? Can you justify why did you choose _not_ to do that?
How would changing this module to a class change your program?

Reply:
no tbh I found pretty to use a module but no use of it at all, I just forgot to remove it at the end, my bad.
I wanted to encapsulate all the previous class in it but I finally did a class. which is calling everything.